finalfrontier/src/config.rs

```diff
@@ -97,6 +97,21 @@ pub struct DepembedsConfig {
     pub untyped: bool,
 }
 
+/// Options for sizing vocabulary.
+#[derive(Copy, Clone, Debug, Serialize)]
+pub enum VocabCutoff {
+    /// Minimum toke count
+    ///
+    /// If MinCount is used, then no word-specific embeddings will be trained for tokens occurring
```

Comment:
Replace by (since it's not only about not training embeddings, the words will also not be used as context):

Discard tokens that occur less than the given count.